vmdb/config/locales/en.yml

```diff
@@ -819,6 +819,7 @@ en:
       EmsCloud:                 Cloud Provider
       EmsCluster:               Cluster
       EmsClusterPerformance:    Performance - Cluster
+      EmsContainer:             Provider
```

Comment:
I think this should say Container Provider, but we can do that in a followup PR.